src/components/ui/drawer/drawer.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode, type FC } from 'react';
+import clsx from 'clsx';
+
+interface DrawerProps {
+    isOpen: boolean
+    children: ReactNode
+    drawerWidth?: number
+    side?: 'left' | 'right'
+}
+
+
+export const Drawer: FC<DrawerProps> = ({ isOpen, children, drawerWidth = 460, side = 'right' }) => {
+
+    return (
+      <div className={
+        clsx(
+          'absolute top-0 h-full',
+          side === 'left' ? 'left-0' : 'right-0'
+        )
+      }
+      style={{ width: `${drawerWidth}px` }}
```

Comment:
**Suggestion**
What if I want to pass width in '%' or 'rem' ? 🤔 Maybe it make sense to change this to be more generic and allow to pass tailwind class instead of absolute value.